test/xpu_api/utilities/pairs/pair.astuple/get_non_const.pass.cpp

```diff
@@ -0,0 +1,62 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/utility>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+class KernelPairTest;
+
+void
+kernel_test()
+{
+    sycl::queue deviceQueue = TestUtils::get_test_queue();
+    bool ret = false;
+    sycl::range<1> numOfItems{1};
+    sycl::buffer<bool, 1> buffer1(&ret, numOfItems);
+    deviceQueue.submit([&](sycl::handler& cgh) {
+        auto ret_access = buffer1.get_access<sycl::access::mode::write>(cgh);
+        cgh.single_task<class KernelPairTest>([=]() {
+            {
+                typedef dpl::pair<int, short> P;
+                P p = dpl::make_pair(3, static_cast<short>(4));
+                ret_access[0] = (dpl::get<0>(p) == 3);
+                ret_access[0] &= (dpl::get<1>(p) == 4);
+                dpl::get<0>(p) = 5;
+                dpl::get<1>(p) = 6;
+                ret_access[0] &= (dpl::get<0>(p) == 5);
+                ret_access[0] &= (dpl::get<1>(p) == 6);
```

Comment:
Can we move this test-case into `get_const_pair_as_tuple.pass.cpp` and remove this test file.

Reply:
Done